packages/richtypo-rules-en/src/en.js

```diff
@@ -0,0 +1,37 @@
+import {
+	spaces,
+	abbr,
+	quotesFactory,
+	numberOrdinalsFactory,
+	numberSeparatorsFactory,
+	numberUnits,
+	amp,
+	emdash,
+	ellipsis,
+} from 'richtypo-rules-common';
+
+const thousandsSeparator = ',';
+const decimalsSeparator = '[.]';
+const ordinals = '(st|nd|rd|th)';
+const openingQuote = '“';
+const closingQuote = '”';
+
+const quotes = quotesFactory({ openingQuote, closingQuote });
+const numberOrdinals = numberOrdinalsFactory({ ordinals });
+const numberSeparators = numberSeparatorsFactory({
+	thousandsSeparator,
+	decimalsSeparator,
+});
+
+const numbers = [numberOrdinals, numberSeparators, numberUnits];
+
+export default {
```

Comment:
Shouldn't we use named exports to be able to do:

```js
import {numbers, all} from 'richtypo-rules-en';
```

Reply:
Yes you're right. Do you think we could have the `all` rule as the default export?

Reply:
Done in ed79b7fd986849dcb484f5449af4cc417a8f130d